cfme/tests/openstack/infrastructure/test_host_lifecycle.py

```diff
@@ -74,7 +77,14 @@ def test_register_host(provider, host, has_mistral_service):
     Metadata:
         test_flag: openstack_scale"""
     hosts_before = [h.uuid for h in provider.mgmt.iapi.node.list()]
-    provider.register(provider.data['instackenv_file_path'])
+
+    """instackenv.json is a file containing the node(s) to be registered.
```

Comment:
**Required**

I disagree with this change, now that I understand exactly what you're designing here. This should absolutely still be pulled from the provider yaml data, and not hard coded within a test function.

Reply:
If anything, a `python.path.local` object should be created in order to verify the file exists before proceeding, and skipping the test if it does not.